sycl/test-e2e/Matrix/SG32/joint_matrix_annotated_ptr.cpp

```diff
@@ -5,15 +5,14 @@
 // SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 //
 //===----------------------------------------------------------------------===//
+// VNNI transform and sub-group size 32 are not supported yet on DG2 by IGC
```

Comment:
VNNI is not relevant for this test
```suggestion
// Sub-group size 32 is not supported yet on DG2 by IGC
```